keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -252,9 +252,7 @@ def test_call_with_custom_label_encoder(self):
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
     def test_backbone_preset(self):
-        for preset in keras_cv.models.RetinaNet.presets:
-            if preset in keras_cv.models.RetinaNet.presets_with_weights:
-                continue
+        for preset in keras_cv.models.RetinaNet.presets_without_weights:
```

Comment:
Maybe now that we don't have this extra logic we can just make this a parameterized test? (This will make error messages much nicer)